.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ orbs:
   circleci-maven-release-orb: sonatype-nexus-community/circleci-maven-release-orb@0.0.15
 
 build-and-test-commands: &build-and-test-commands
-  mvn-build-test-command: mvn clean verify -PbuildKar -Dit
+  mvn-build-test-command: mvn clean --batch-mode verify -PbuildKar -Dit
```

Comment:
minor question: why was this `--batch-mode` argument needed? Was there some error occurring? If so, I'd like to understand more, and possible propagate this change to other projects if needed. If not, maybe this change should be reverted?

Reply:
To avoid this kind of [log](https://circleci.com/api/v1.1/project/github/sonatype-nexus-community/nexus-repository-apk/103/output/103/0?file=true&allocation-id=5f7b1c516712c17dd216a0f8-0-build%2F7BFDEC69) when building a project. To me download progression bars are useless when builds are managed by a CI platform and logs like [this](https://circleci.com/api/v1.1/project/github/sonatype-nexus-community/nexus-repository-apk/109/output/103/0?file=true&allocation-id=5f7c792ae9785b19b63ae954-0-build%2F371272BB) are more human readable, aren't they?

Reply:
Ah, cool! I was concerned there was some failure occurring. Squelching CI build noise is an excellent reason.

Perhaps you'd like to add a similar change to our [Nexus Format Archetype](https://github.com/sonatype-nexus-community/nexus-format-archetype/blob/master/src/main/resources/archetype-resources/.circleci/config.yml#L17) ?